docker/centos-arm64.ks

```diff
@@ -0,0 +1,134 @@
+install
+keyboard us
+network  --bootproto=dhcp --device=eth0 --onboot=on
+rootpw --iscrypted $1$UKLtvLuY$kka6S665oCFmU7ivSDZzU.
+timezone UTC --isUtc 
+selinux --enforcing
+firewall --disabled
+#repo --name="CentOS" --baseurl=http://mirror.centos.org/centos/7/os/x86_64/ --cost=100
+#repo --name="CentOS" --baseurl=ftp://192.168.3.1/centos7forARM/   --cost=100
+repo --name="CentOS" --baseurl=http://buildlogs.centos.org/centos/7/os/aarch64/   --cost=100
+# here you must download from my shared rpm package,and create a localrepo . and MAKEDEV rpm package
+repo --name="systemdcontainer" --baseurl=ftp://192.168.3.1/systemd-container/ --cost=100
```

Comment:
Because build aarch64 image needs systemd-container rpm package , and there are no package in any website , and i compile it locally , so if you need to build the docker aarch64 image , you must createrepo in local and set the repository.